src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String SET_LOGIN = "validLogin";
+    private static final String SET_PASSWORD = "validPass";
```

Comment:
```suggestion
    private static final String VALID_LOGIN = "validLogin";
    private static final String VALID_PASSWORD = "validPass";
```

you can also add VALID_AGE constant